test/parallel/test-net-connect-handle-econnrefused.js

```diff
@@ -27,11 +27,10 @@ const assert = require('assert');
 const server = net.createServer();
 server.listen(0);
 const port = server.address().port;
-const c = net.createConnection(port);
-
-c.on('connect', common.mustNotCall());
-
-c.on('error', common.mustCall((e) => {
-  assert.strictEqual('ECONNREFUSED', e.code);
-}));
-server.close();
+server.close(() => {
```

Comment:
Can you wrap this in `common.mustCall`?

Reply:
@joyeecheung It would be nice. I'll fix it.